pygmt/src/wiggle.py

```diff
@@ -65,14 +77,12 @@ def wiggle(self, data=None, x=None, y=None, z=None, **kwargs):
         **+w**\ *length*\ [**+j**\ *justify*]\ [**+al**\|\ **r**]\
         [**+o**\ *dx*\ [/*dy*]][**+l**\ [*label*]].
         Defines the reference point on the map for the vertical scale bar.
-    color : str
-        Set fill shade, color or pattern for positive and/or negative wiggles
-        [Default is no fill]. Optionally, append **+p** to fill positive areas
-        (this is the default behavior). Append **+n** to fill negative areas.
-        Append **+n+p** to fill both positive and negative areas with the same
-        fill. **Note**: You will need to repeat the color parameter to select
-        different fills for the positive and negative wiggles.
-
+    fillpositive : str
+        Set fill shade, color or pattern for positive wiggles [Default is no
+        fill].
+    fillnegative : str
+        Set fill shade, color or pattern for negative wiggles [Default is no
+        fill].
```

Comment:
```suggestion
    fillpositive : str
        Set fill shade, color, or pattern for positive wiggles [Default is no
        fill].
    fillnegative : str
        Set fill shade, color, or pattern for negative wiggles [Default is no
        fill].
```

Reply:
Have we agreed to use this Oxford comma in all documentations? Both versions are correct but we should have a consistent style across all docs.

Reply:
At the moment I do not remember that there was a discussion about this in detail. I feel that currently both versions are used in the documentation. Here, I orientated myself on the version used for the `scale` parameter above, and there a comma is set before the `or`. I have no strong preference on this, but consistency would  be good.